src/components/Section/Section.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import SectionCard from '../../Shared/SectionCard/SectionCard';
+
+
+function Section() {
+  return (
+      <>
+    <div class="section">
+    <h2>New Arrivals</h2>
+
+    <div class="newArrival__carasoul">
+    
+    <SectionCard src='/assets/landing page Images/pngwing.com.png' name='Heisenberg Blue Finish'  price={9999}/>
+    
+    <SectionCard src='/assets/landing page Images/category watch 2 .png' name='George Jensen white finish'  price={9999}/>
+    
+    <SectionCard src='/assets/landing page Images/category watch 3 .png' name='Rado ferrari finish'  price={9999}/>
+    
+    <SectionCard src='/assets/landing page Images/category watch 4 .png' name='Hubolt sailor finish'  price={9999}/>
+
```

Comment:
you can store this in array-like newarrivalsItem=[{ image:"./assets/imag.jpg, name:"Heisenberg Blue Finish"},{ image:"./assets/imag.jpg, name:"Heisenberg Blue Finish"}] then map on it